recipes/cpython/all/conanfile.py

```diff
@@ -161,7 +161,7 @@ def requirements(self):
             if tools.Version(self.version) < "3.8":
                 self.requires("mpdecimal/2.4.2")
             else:
-                self.requires("mpdecimal/2.5.1")
+                self.requires("mpdecimal/2.5.0  ")
```

Comment:
Space

Reply:
Little typo..
Thanks for the double check.